tests/integration-tests/tests/configure/test_pcluster_configure.py

```diff
@@ -161,3 +238,14 @@ def assert_config_contains_expected_values(
             validator.get("parameter_name")
         )
         assert_that(observed_value).is_equal_to(validator.get("expected_value"))
+
+
+@pytest.fixture()
+def subnet_in_use1_az3(vpc_stack):
+    # use1_az3 does not have c5.xlarge or m6g.xlarge
```

Comment:
Can we do an assertion somewhere to assert this initial condition? It is possible that c5.xlarge will get added to use1_az3 in the future and we won't be aware of the issue